metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/objectImporters/ObjectsImporter.java

```diff
@@ -18,18 +18,21 @@
 
 package uk.ac.ebi.ampt2d.metadata.importer.objectImporters;
 
-import uk.ac.ebi.ampt2d.metadata.importer.objects.AnalysisObject;
-import uk.ac.ebi.ampt2d.metadata.importer.objects.ReferenceSequenceObject;
-import uk.ac.ebi.ampt2d.metadata.importer.objects.SampleObject;
-import uk.ac.ebi.ampt2d.metadata.importer.objects.StudyObject;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Analysis;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.ReferenceSequence;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Sample;
+import uk.ac.ebi.ampt2d.metadata.persistence.entities.Study;
+
+import java.util.List;
+import java.util.Set;
 
 public interface ObjectsImporter {
```

Comment:
I'm not sure there is any benefit in having this interface. I would just leave the concrete class and remove the subpackage, because it will contain just said class.

Reply:
ya because I am going to introduce another importer for reading xmls from database in analysis import (more specifically for EGA), because the study xmls from the database doesn't contain analysis link, so we need to go via analysis in that case.